src/readers/morphologyASC.cpp

```diff
@@ -152,32 +151,41 @@ class NeurolucidaParser
       Add the last point of parent section to the beginning of this section
       if not already present.
 
-     The idea is that these two structures should represent the same morphology:
-
-     (3 -8 0 2)     and          (3 -8 0 2)
-     (3 -10 0 2)                 (3 -10 0 2)
-     (                           (
-       (0 -10 0 2)                 (3 -10 0 2)  <-- duplicate parent point
-       (-3 -10 0 2)                (0 -10 0 2)
-       |                           (-3 -10 0 2)
-       (6 -10 0 2)                 |
-       (9 -10 0 2)                 (3 -10 0 2)  <-- duplicate parent point
-     )                             (6 -10 0 2)
-                                   (9 -10 0 2)
-                                 )
+      The diameter is taken from the child section next point as does NEURON.
+      Here is the spec:
+      https://bbpteam.epfl.ch/project/issues/browse/NSETM-1178?focusedCommentId=135030&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-135030
+
+      In term of diameters, the result should look like the right picture of:
+      https://github.com/BlueBrain/NeuroM/issues/654#issuecomment-332864540
+
+      The idea is that these two structures should represent the same morphology:
+
+      (3 -8 0 2)     and          (3 -8 0 2)
```

Comment:
I would use non-constant diameters here to make the point clear for users nto reading this PR.

Reply:
Very true !